docs/dataset/advanced-usage/set-up-natural-language-search.md

```diff
@@ -0,0 +1,157 @@
+---
+icon: kolena/comparison-16
+---
+
+# :kolena-comparison-20: Natural Language Search Setup
+
+Kolena supports natural language and similar image search
+across [`Image`][kolena.asset.ImageAsset] data registered to the platform.
```

Comment:
I would keep this unlinked — the primary use case is for image datapoints, which doesn't have a clear place to link at this time.